AdminWebsite/AdminWebsite/ClientApp/src/app/suitability/services/questionnaire.service.ts

```diff
@@ -5,32 +5,31 @@ import { ScrollableSuitabilityAnswersService } from './scrollable-suitability-an
 export class QuestionnaireResponses {
   readonly items: ParticipantQuestionnaire[];
   readonly hasMore: boolean;
+  readonly nextCursor: string;
 
-  constructor(items: ParticipantQuestionnaire[], hasMore: boolean) {
+  constructor(items: ParticipantQuestionnaire[], hasMore: boolean, nextCursor: string) {
     this.items = items;
     this.hasMore = hasMore;
+    this.nextCursor = nextCursor;
   }
 }
 
 @Injectable()
 export class QuestionnaireService {
-  private nextCursor = '';
-  private hasMore = true;
 
   constructor(private service: ScrollableSuitabilityAnswersService) { }
 
-  async loadNext(): Promise<QuestionnaireResponses> {
-    if (!this.hasMore) {
-      return new QuestionnaireResponses([], false);
+  async loadNext(hasMore: boolean, nextCursor: string): Promise<QuestionnaireResponses> {
+    if (!hasMore) {
```

Comment:
It looks like `hasMore` is now handled outside of this class so maybe we can move that out to the caller or is there a reason to have it inside here?

Reply:
Yes, I think we don't need it any more in the service. I moved next cursor and hasMore to component as it was a reason for duplication, service was keeping value of this variables and don't care about component actions.